forms-html/yazen_lawin/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+<div style="font-weight: bold;">
+
+
+
+<form>
+    <label for="email"> Enter your email:</label> <br>
+    <input type="email" id="email" name="email" value="osama@recoded.com" required > <br>
+    <label for="username">user name:</label><br>
+    <input type="text" id="username" name="username"  value="osama" required ><br>
+    <label for="password">password:</label><br>
+    <input type="password" id="password" name="password"  value="12458" required ><br>
+    <label for="quantity">quantity(between 1 and 5):</label>
+    <input type="Number" id="quantity" name="quantity" min="1" max="5" required ><br>
+    <label for="volume">Volume(between 0 and 50):</label>
+    <input type="range" id="volume" name="volume" min="0" max="50" required ><br>
+    <br>
+    <label for="submit">radio</label><br>
+    <br>
+    <input type="radio" id="male" name="gender" value="male">
+    <label for="male">Male</label><br>
+    <input type="radio" id="female" name="gender" value="female">
+    <label for="female">Female</label><br>
+    <input type="radio" id="other" name="gender" value="other">
+
+    <label for="other">Other</label><br>
+    <br>
+
+    <label for="submit">checkbox</label><br>
+    <br>
+    <input type="checkbox" id="vehicle 1" name="vehicle1" value="bike">
+    <label for="male">Ihave a bike</label><br>
+    <input type="Checkbox" id="vehicle 2" name="vehicle2" value="car">
+    <label for="checkbox">I have a car</label><br>
+    <input type="Checkbox" id="vehicle 3" name="vehicle3" value="boat">
+    <label for="checkbox">I have a boat</label>
+    <br>
+    <br>
+<label for="submit">button</label><br>
+<br>
+<input type="submit" id="submit" value="click me"><br>
+
+<label for="favoraite color">Select your favorite color:</label>
+<input type="color" id="favoraite color" name="favoraite color" value="black"><br>
+<br>
+<label for="date">Enter a date before 1980-01-01:</label>
+<input type="date" id="date" name="date" max="1980-01-01">
+<p>date and time:</p>
+<label for="birthday(date and time)">birthday(date and time):</label>
+<input type="datetime-local" id="birthday(date and time)" name="birthday(date and time)" value="04/15/2020, 04:12 PM">
+<br>
+<br>
+<label for="submit">submit button</label><br>
+<br>
+<input type="submit" id="submitdata" value="submit"><br>
+<br>
+<label for="reset">reset from button </label><br>
+<br>
+<input type="submit" id="reset" value="reset"><br>
```

Comment:
the type pf reset button should be **reset**

```suggestion
<input type="reset" id="reset" value="reset"><br>

```